.github/ISSUE_TEMPLATE/feature-suggestion.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Feature Suggestion
+about: Suggest a feature to improve this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- 
+Thanks for suggesting a feature for the NUnit Console/Engine. NUnit is run entirely by volunteers from the community, so we welcome pull requests and contributions in all areas.
+
+Please describe what you're proposing, and let us know if you'd be willing to submit a pull request as well. It's best to wait for a response from the team before starting to write any code - we want to be sure any new features fit in with the overall vision for NUnit, and aren't going to cause any breaking changes.
```

Comment:
Nitpick. I don't think that there should be a comma in `"we want to be sure any new features fit in with the overall vision for NUnit, and aren't going to cause any breaking changes"` as there is no subject in the clause after the comma. But the resulting sentence becomes a bit long, perhaps we can rephrase it?

Reply:
Break sentence after "code"? Agree about the comma.